packages/kit/src/runtime/client/client.js

```diff
@@ -1146,6 +1146,8 @@ export function create_client(app, target) {
 			reset_focus();
 		}
 
+		console.log(document.activeElement);
```

Comment:
This should be removed.

It also looks like some tests are failing.

Reply:
Thanks for catching that! From the tests I found out that we needed to avoid changing the scroll position too when using `location.replace()`. It restores the previous scroll position now.